### PR TITLE
Add missing ReadTheDocs sections

### PR DESCRIPTION
**Pull request by pmantica1**
I thoroughly compared the README.rst documentation and the ReadTheDocs documentation and found that these sections were outright missing from ReadTheDocs but were included in the README.rst.

### CODE REVIEW

File: docs/source/language_specification/schema_types.rst [hunk 1]
```diff
@@ -316,6 +316,13 @@ In this query, the :code:`out_Entity_Related` is of :code:`Entity` type.
 However, the query only wants to return results where the related entity
 is a :code:`Species`, which :code:`... on Species` ensures is the case.
 
+Constraints and Rules
```
Comment (pmantica1) [inline docs/source/language_specification/schema_types.rst:319]: This section and the one below are purely copy paste.

File: README.rst [hunk 2]
```diff
@@ -1443,7 +1443,7 @@ mutate, the alternative approach is via the
 :code:`insert_meta_fields_into_existing_schema()` helper function defined by
 the compiler:
 
-::
+.. code:: python
```
Comment (pmantica1) [inline README.rst:1446]: Adding this syntax annotation.

**Review by chewselene — APPROVED** [review]
looks good to me!

**Comment by pmantica1** [comment]
For some reason when I copied the text from github it did not include some line breaks. I thought that I added all of them but I missed one. So CI failed because the rst text was "invalid" without that newline. I added the missing newline and tested that the sphinx CI command works correctly.

**Review by chewselene — APPROVED** [review]

**Pull request merged**